CMakeLists.txt

```diff
@@ -171,6 +171,13 @@ if(NVCC_THREADS AND VLLM_GPU_LANG STREQUAL "CUDA")
   list(APPEND VLLM_GPU_FLAGS "--threads=${NVCC_THREADS}")
 endif()
 
+#
+# Set nvcc fatbin compression.
+#
+if(${CMAKE_CUDA_COMPILER_VERSION} VERSION_GREATER 12.8 AND VLLM_GPU_LANG STREQUAL "CUDA")
```

Comment:
12.7 probably works too, untested. And 12.6 almost works too in PyTorch, we just hit a compiler bug with libtorch. It's supported since 12.4 technically.

Reply:
Since our default CI and wheel is CUDA 12.8, it felt most safe to just test this for now.